app/src/main/java/org/fossasia/susi/ai/helper/CredentialHelper.java

```diff
@@ -58,38 +62,20 @@ public static void clearFields(TextInputLayout... layouts) {
      * Check password valid boolean.
      *
      * @param password the password
-     * @param context  the context
      * @return the boolean
      */
-    public static boolean checkPasswordValid(TextInputLayout password, Context context) {
-        if (password.getEditText() == null)
-            throw new IllegalArgumentException("No Edittext hosted!");
-        if (!isPasswordValid(password.getEditText().getText().toString())) {
-            password.setError(context.getString(R.string.pass_validation_text));
-            return false;
-        } else {
-            password.setError(null);
-            return true;
-        }
+    public static boolean checkPasswordValid(String password) {
```

Comment:
Why making new method to call another method which does the same thing? Remove checkPasswordValid and use isPasswordValid when calling.